crates/runtime/src/pallets.rs

```diff
@@ -158,6 +160,16 @@ parameter_types! {
     pub const InvokeTxMaxNSteps: u32 = 1_000_000;
     pub const ValidateMaxNSteps: u32 = 1_000_000;
     pub const ProtocolVersion: u8 = 0;
+    // Starknet testnet SN_GOERLI
+    // Need to use `from_mont` because this needs to be a constant function call
+    pub const ChainId: Felt252Wrapper = Felt252Wrapper(FieldElement::from_mont(
+        [
+            3753493103916128178,
+            18446744073709548950,
+            18446744073709551615,
+            398700013197595345,
+        ]
+    ));
```

Comment:
can you make this `Felt252Wrapper(FieldElement::from_mont(
        [
            3753493103916128178,
            18446744073709548950,
            18446744073709551615,
            398700013197595345,
        ]
    ))` a constant named `SN_GOERLI_CHAIN_ID`